README.rst

```diff
@@ -21,19 +21,7 @@ Documentation
 =============
 Up to date documentation:
 
-- `See the wiki <https://github.com/ultrabug/py3status/wiki>`_
-
-Learn how to easily **handle i3bar click events** directly from your i3status config:
-
-- `Handle click events directly from your i3status config <https://github.com/ultrabug/py3status/wiki/Handle-click-events-directly-from-your-i3status-config>`_
-
-Learn how to **extend your current i3status config** to easily interact with your i3bar:
-
-- `Load and order py3status modules directly from your current i3status config <https://github.com/ultrabug/py3status/wiki/Load-and-order-py3status-modules-directly-from-your-current-i3status-config>`_
-
-Learn how to write your own modules:
-
-- `Write your own modules <https://github.com/ultrabug/py3status/wiki/Write-your-own-modules>`_
+- `Usage, writing your own modules etc <https://github.com/ultrabug/py3status/blob/master/doc/README.md>`_
```

Comment:
I know I'm picky, but I liked the actual list of possibilities in the main README instead of a catch-all link.

This gives new comers a clear view of the cool possibilities the project offers and is a great incentive to try it out :)